src/WindowsUtils/DllInjectionTest.cpp

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <absl/strings/ascii.h>
+#include <absl/strings/str_format.h>
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+#include <windows.h>
+
+#include "OrbitBase/Logging.h"
+#include "OrbitBase/ThreadUtils.h"
+#include "Test/Path.h"
+#include "TestUtils/TestUtils.h"
+#include "WindowsUtils/DllInjection.h"
+
+namespace orbit_windows_utils {
+
+namespace {
+
+[[nodiscard]] std::filesystem::path GetTestDllPath() {
+  return orbit_test::GetTestdataDir() / "libtest.dll";
+}
+
+[[nodiscard]] std::filesystem::path GetNonExistentDllPath() { return "Z:/non_existent_dll.dll"; }
+
+}  // namespace
+
+TEST(Injection, InjectDllInCurrentProcess) {
+  // Injection.
+  uint32_t pid = orbit_base::GetCurrentProcessId();
+  ErrorMessageOr<void> result = InjectDll(pid, GetTestDllPath());
+  if (result.has_error()) ERROR("Running InjectDllInCurrentProcess: %s", result.error().message());
+  EXPECT_FALSE(result.has_error());
```

Comment:
You should use `EXPECT_THAT(...result, orbit_test_utils::HasNoError())`. That way, in case there is an error, it prints it, without the need to do it manually.